pages/mesosphere/dcos/version-policy/index.md

```diff
@@ -406,13 +406,13 @@ The designations are as follows:
         <td><p style="text-align: center;">⚫</p></td>
     </tr>
     <tr>
-        <td>Kafka 2.4.x-1.1.1</td>
+        <td>Kafka 2.8.x-2.3.0</td>
```

Comment:
We _don't_ want to change this value because we want to support n-1 versions of the _base_ tech not n-1 of our DCOS packages. Here we want to communicate to customers that we support Kafka 1 and Kafka 2.

Reply:
Agreed! Will fix this also.